app/assets/stylesheets/course/layout.scss

```diff
@@ -1,24 +1,42 @@
 .course-layout {
   @include timestamp;
 
-  #course-user-badge {
-    #course-user-progress {
-      .progress {
-        margin-bottom: 10px;
-        margin-top: 10px;
-
-        .course-user-experience-points {
-          min-width: 10em;
-        }
-      }
+  #course-badge-achievement {
+    margin-bottom: 0.5em;
+    margin-left: 0.5em;
+
+    .image > img {
+      height: $course-user-badge-achievement-badge;
+      width: $course-user-badge-achievement-badge;
+    }
+
+    .achievement > a:hover {
+      text-decoration: none;
+    }
+
+    .achievement-difference {
+      margin-left: 6px;
+    }
+  }
+
+  #course-badge-level {
+    margin-left: 0.5em;
+
+    .level {
+      font-size: 150%;
+      font-weight: 300;
```

Comment:
Personally feel that this should be normal font weight, or even slightly bolded. Even though the level is bigger, the EXP count looks like the focal point of that line because it is relatively bold and it is nearer the center:

<img width="258" alt="image" src="https://user-images.githubusercontent.com/6252919/40294484-5d31add8-5d08-11e8-8d73-25825f3118a7.png">


Since there is a font size difference, I think it is okay to remove the parentheses to make it cleaner; visually, they are quite separate already. 

Also, since the EXP count is the less important gamification statistic, am okay to let it be less bold than normal.